### PR TITLE
Docs: update dispose return type

### PR DESCRIPTION
**Pull request by servinlp**
**Description**

On multiple places change the documentation for the `dispose()` method to return null

### CODE REVIEW

**Review by Mugen87 — APPROVED** [review]

**Comment by mrdoob** [comment]
I think it should be `[method:undefined dispose]`.

**Comment by Mugen87** [comment]
Then it's best if the OP corrects the other methods, too. `null` is used for all `void` helper methods right now.

**Comment by mrdoob** [comment]
Oh... I guess we have many incorrect ones like that?

**Comment by servinlp** [comment]
Ye `method:null` is currently used on quite a few places. Should all void methods have this as there return type? Like for example [OrthographicCamera.setViewOffset](https://threejs.org/docs/index.html?q=ort#api/en/cameras/OrthographicCamera.setViewOffset) ([the code](https://github.com/mrdoob/three.js/blob/master/src/cameras/OrthographicCamera.js#L44))

**Comment by Mugen87** [comment]
Yes, the docs are currently not consistent regarding returns types. If a method has not `return` statement, you can find the following approaches in the docs:

- Using `void`:
https://github.com/mrdoob/three.js/blob/efd64661632ce3078601553ac5e59c0cf4618a8c/docs/api/en/loaders/Loader.html#L65

- Using `null`:
https://github.com/mrdoob/three.js/blob/efd64661632ce3078601553ac5e59c0cf4618a8c/docs/api/en/helpers/CameraHelper.html#L76

- Using `undefined`:
https://github.com/mrdoob/three.js/blob/efd64661632ce3078601553ac5e59c0cf4618a8c/docs/api/en/renderers/WebGLCubeRenderTarget.html#L68

I think I vote for using `void` since this type is also used in TS.

**Comment by servinlp** [comment]
`void` sound logical from a TS point of view however from having a quick look around it seems that `method:null` is most used right now.

**Comment by Mugen87** [comment]
Before using `null` I really should be `undefined`.

**Comment by mrdoob** [comment]
These methods do not return `null`. Either `void` or `undefined` works for me.

**Comment by servinlp** [comment]
So `void`? I'm assuming we would want to do this them throughout the entire docs in the future aswell?

**Comment by Mugen87** [comment]
Then let's use `undefinde` :+1:.

@servinlp Are you willing to update the docs accordingly?

**Comment by servinlp** [comment]
I will update this PR first and then see if I can change the rest in a different PR.

Do we have a document or something were we define documentation guide lines?

**Comment by Mugen87** [comment]
> Do we have a document or something were we define documentation guide lines?

No, if possible I would like to avoid that and just correct the existing doc pages. If everything is consistent, copy/paste errors like before can be avoided.

File: docs/api/en/helpers/CameraHelper.html [hunk 1]
```diff
@@ -12,40 +12,35 @@
...
 		</p>
 
 		<h2>Code Example</h2>
 		<code>
-const camera = new THREE.PerspectiveCamera( 75, window.innerWidth / window.innerHeight, 0.1, 1000 );
-const helper = new THREE.CameraHelper( camera );
-scene.add( helper );
+			const camera = new THREE.PerspectiveCamera( 75, window.innerWidth /
+			window.innerHeight, 0.1, 1000 ); const helper = new THREE.CameraHelper(
+			camera ); scene.add( helper );
```
Comment (Mugen87) [inline docs/api/en/helpers/CameraHelper.html:23]: Please undo these changes. The code in this block should be like that. Otherwise the formatting will be odd.

https://raw.githack.com/servinlp/three.js/feature/dispose-docs/docs/index.html#api/en/helpers/CameraHelper

Comment (Mugen87) [reply docs/api/en/helpers/CameraHelper.html:23]: In general, please avoid any other formatting changes. It's best to focus on just the return types.

Comment (servinlp) [reply docs/api/en/helpers/CameraHelper.html:23]: Sorry about that, though I had it turned off. Will correct it.

**Comment by Mugen87** [comment]
Do you mind updating the Chinese pages, too?

**Comment by servinlp** [comment]
It looks like the Chinese pages don't have the documentation for the dispose method yet

**Review by Mugen87 — APPROVED** [review]

**Comment by mrdoob** [comment]
Thanks!
